plugins/foreman_cockpit/1.0/index.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: plugin
+title: Foreman Cockpit 1.0 manual
+version: 1.0
+---
+
+# 1. {{ page.title }}
+
+[Foreman Cockpit](https://github.com/theforeman/foreman_cockpit) allows you to see a hosts' Cockpit in Foreman.
+
+[Cockpit](http://cockpit-project.org/) is a lightweight service that runs
+in your hosts, and allows you to perform sysadmin tasks through a web UI.
+
+Integration between Cockpit and Foreman is seamless. Foreman automatically
+detects which of your hosts are running Cockpit.
+
+![gif demoing foreman_cockpit](https://camo.githubusercontent.com/dd327f958993d60ecefbe4e1fc26bc45ad63cf9c/687474703a2f2f692e696d6775722e636f6d2f527a64735239622e676966)
+
+# 2. Installation
+
+### 2.1.1 Installer (recommended)
+
+As of Foreman 1.11+, the foreman-installer is able to automatically install the
+package and proceed with all necessary steps. To do this, re-run the installer
+with the following option:
+
+    # foreman-installer --enable-foreman-plugin-cockpit
+
+Important note: Executing foreman-installer will re-deploy all foreman-related
+configuration files. In case changes have been made, proceed with manual
+installation.
+
+### 2.1.2 Manual installation
+
+If this is the first plugin you're installing, please see the [plugin
+repository section](plugins/#2.2Packageinstallation) to set up the repository
+first.
+
+To install Foreman Cockpit, do the following
+
+    # yum install tfm-rubygem-foreman_cockpit
+
+on Fedora and Red Hat systems or
+
+    # apt-get install ruby-foreman-cockpit
+
+on Debian and Ubuntu systems.
+
```

Comment:
> For RH based systems (BTW, do we ship Feldora packages at the moment?) do

We do, but their name would be "rubygem-foreman_cockpit", not "ruby-foreman-cockpit" - identical to EL, without the prefix.